spec/unit/provider/package/yum_spec.rb

```diff
@@ -194,5 +194,28 @@
         expect(output).not_to include("Random plugin")
       end
     end
+
+    context "with subscription manager enabled " do
+      let(:check_update) do
+        File.read(
+          my_fixture("yum-check-update-subscription-manager.txt")
+        )
+      end
+      let(:output) { described_class.parse_updates(check_update) }
+
+      it "parses correctly formatted entries" do
+        expect(output['curl.x86_64']).to eq(
+          [
+            {
+              :name => 'curl',
+              :epoch => '0',
+              :version => '7.32.0',
+              :release => '10.fc20',
+              :arch => 'x86_64'
+            }
+          ]
+        )
```

Comment:
Just for consistency, can you do this in one line like the tests above?

Reply:
I can, i just don't like to scroll sideways when looking at the code on github 😓